### ISSUE TITLE
how do I use it in SPFx react

### ISSUE THREAD

**Issue by Ofer-Gal** [anomepani/sp-rest-util#5]
I tried to import just BatchUtils but it does not work.  So I change it to "export const BatchUtils = (() => { ..." and it worked 
The tslinter insisted on having something for "FormDigestValue" even though you don't need it for GET
So now 2 URLs that work fine in arr bring back an empty result with no error.
BatchUtils.GetBatchAll({ rootUrl: window.location.origin, batchUrls: arr, FormDigestValue: null })

Any idea how to fix?

**Comment by anomepani** [comment]
@Ofer-Gal , you are right need to use export const If you need to import with spfx react.

`FormDigestValue` not required for GET Request but `batch` API is post request so 
`FormDigestValue` value will be use but if you do not pass digest it will internally generate digest based on given parameter.

Could you please attach request and payload and response from network tab, it is helpful for troubleshoot issue.
